src/App.js

```diff
@@ -1,23 +1,70 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { useState } from "react";
+import PersonInfo from "./components/PersonInfo";
+import GithubInfo from "./components/GithubInfo";
+
+const GITHUB_API = "https://api.github.com/users/";
 
 function App() {
+  const [username, setUserName] = useState("");
+  const [userData, setUserData] = useState("");
+  const [err, setErr] = useState(false);
+  const [loading, setLoading] = useState(true);
+  const [initialMsg, setInitialMsg] = useState(true);
```

Comment:
We don't need this extra state. Try to implement this same functionality without this state.